test/e2e/mnist_pytorch_appwrapper_test.go

```diff
@@ -145,6 +147,10 @@ func runMnistPyTorchAppWrapper(t *testing.T, accelerator string, numberOfGpus in
 		},
 	}
 
+	raw, _ := json.Marshal(job)
+	// Workaround to remove creationTimestamp which is brought by json marshaller, this field breaks Kueue reconciliation
+	patchedRaw := strings.ReplaceAll(string(raw), `"metadata":{"creationTimestamp":null},`, "")
```

Comment:
Could this be encapsulated into the `Raw` helper function?

Reply:
I can add it there as an option

Reply:
Added the replace into dedicated helper function.
I didn't want to add it to the `Raw` as the manipulation of the content needs to be done on a struct pointer, so in the end I have manipulated with `RawExtension` anyway. In such case `RawExtension` can be modified outside of `Raw`.